pkg/metadata/util.go

```diff
@@ -16,10 +16,22 @@ limitations under the License.
 
 package metadata
 
-import "github.com/kubernetes-incubator/cri-containerd/pkg/metadata/store"
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-containerd/pkg/metadata/store"
+)
+
+// netNSFormat is the format of network namespace of a process.
+const netNSFormat = "/proc/%v/ns/net"
 
 // IsNotExistError is a helper function to check whether the error returned
 // by metadata store is not exist error.
 func IsNotExistError(err error) bool {
 	return err.Error() == store.ErrNotExist.Error()
 }
+
+// getNetworkNamespace returns the network namespace of a process.
+func getNetworkNamespace(pid uint32) string {
```

Comment:
We should use the permanent network namespace path.

One of the reason we need permanent path is that, once the sandbox process is gone, you will not be able to access `/proc/pid/ns/net`, which is not what we want.

We want to cleanup the network namespace even though the sandbox exits itself. That's why we want to have a known permanent network namespace.